### DISCUSSION TITLE
Not able to set icon using ARPProductIcon property

### DISCUSSION THREAD

**Discussion by navinnarra** [wixtoolset/issues#7334] [Questions]
I am trying Wix V4 toolset. I am able to build the msi, but the icon is not showing up in Programs. 

This is my snippet

```
<Package .. >
                <Icon Id="AppIcon" SourceFile="icon.ico" />
		<Property Id="ARPPRODUCTICON" Value="AppIcon" />
</Package>
```

Not sure what I am missing.

**Comment by robmen** [comment]
Turns out I did just that in [Episode 24 of the Deployment Dojo](https://youtube.com/live/1kV4gk3tTzg?feature=share). You can see it working there.
